router/group.go

```diff
@@ -76,48 +75,86 @@ func (h *Handlers) GetGroups(c echo.Context) error {
 }
 
 func (h *Handlers) PostGroup(c echo.Context) error {
```

Comment:
↓みたいな感じでコメントを残しておいてくれるとわかりやすくて嬉しい
```suggestion
// PostGroup POST /groups
func (h *Handlers) PostGroup(c echo.Context) error {
```